qiskit_ibm_runtime/options/environment_options.py

```diff
@@ -52,12 +52,11 @@ class EnvironmentOptions:
         Default: ``None``.
     """
     private: Optional[bool] = False
-    r"""Boolean that indicates whether the job is marked as private. This is only
```

Comment:
Removing `This is only supported for ``ibm_quantum`` channel.` is premature until the POST /jobs API supports the private field.

The changes on the server side are being worked such that POST /jobs support is the last thing to be supported and that enables the rest of the feature in the other GET APIs.